frontend/src/app/modules/posts/components/comment-delete/comment-delete.component.html

```diff
@@ -0,0 +1,28 @@
+<div *ngIf="visible" class="tooltip tooltip-bottom cursor-pointer -top-4" data-tip="Delete">
```

Comment:
zobacz czy da się tutaj jakoś łatwo ograniczyć duplikację kodu, wydaje mi się że to jest praktycznie to samo co przy usuwaniu posta